src/styles/Blocks/star.scss

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-size: auto;
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    border-radius: 0.5px 0 0;
```

Comment:
The border-radius property seems to be set to '0.5px 0 0', which might not have a visible effect. Consider reviewing if this is necessary or if it should be adjusted.